README.md

```diff
@@ -4,3 +4,4 @@
 
 ## Prerequisites
 
+A humble edit, to prove that I exist.
```

Comment:
I would argue this proves nothing about your existence, but this code is accurate.